sql/chunk.sql

```diff
@@ -27,23 +27,57 @@ $BODY$
 DECLARE
     partition_epoch_row _timescaledb_catalog.partition_epoch;
     chunk_row           _timescaledb_catalog.chunk;
-    time_interval BIGINT;
+    time_interval       BIGINT;
+    start_distinct_ctn  BIGINT;
+    end_distinct_ctn    BIGINT;
 BEGIN
+
     SELECT pe.*
     INTO partition_epoch_row
     FROM _timescaledb_catalog.partition p
     INNER JOIN _timescaledb_catalog.partition_epoch pe ON (p.epoch_id = pe.id)
     WHERE p.id = partition_id
     FOR SHARE;
 
+    -- Check if there are chunks in other partitions.
+    -- If so, use their time range to keep chunks time aligned
+
+    SELECT COUNT(DISTINCT(start_time)), COUNT(DISTINCT(end_time))
```

Comment:
I am not entirely sure what this is testing for. Shouldn't we just test whether this returns 0 or 1 row and act accordingly? Can't we use INTO STRICT for that? I am confused by the DISTINCT stuff